README.md

```diff
@@ -11,9 +11,16 @@
 - [Stephane Reiss](https://github.com/T0RAT0RA)
 - [Jeremie Bardon](https://github.com/jeremiebardon)
 
-## Development
+## Satus
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/e2424f31-4626-483e-8e1c-e558c921c6d5/deploy-status)](https://app.netlify.com/sites/vuemontreal/deploys)
 
-
 # Check our projects if you want to know more about what next for vue montreal
+
+## Dev
+
+`npm ci && npm run dev`
+
+go to [localhost](http://localhost:3000)
+
+you can have access to our [stories](http://localhost:3000/.stories)
```

Comment:
Est-ce que https://deploy-preview-157--vuemontreal.netlify.app/.stories  devrait fonctionner?

Reply:
non c'est normal la 404

Reply:
Ok, I see here: https://github.com/richardeschloss/nuxt-stories#configuration

We would have to set `forceBuild: true` to get those on netlify.

Would it be useful to have this ran on something like https://www.chromaticqa.com/ for visual regressions?

Reply:
we can create a new ticket for adding / merge to chroma yeah. They have a free plan now